gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -119,12 +738,1491 @@ gum_stalker_add_call_probe (GumStalker * self,
+  g_print ("\ttarget.origin_ip = insn->end: %p\n", target.origin_ip);
+#endif
+
+  if (insn->ci->id == ARM64_INS_BL || insn->ci->id == ARM64_INS_B)
+  {
+    g_assert (op->type == ARM64_OP_IMM);
+
+#if DEBUG_LEVEL == 10
+    g_print ("gum_exec_block_virtualize_branch_insn - BL & %sconditonal B\n",
+        is_conditional ? "" : "in");
+    g_print ("\top->imm: %p\n", op->imm);
+    g_print ("\tinsn->ci->address: %p\n", insn->ci->address);
+#endif
+
+    target.absolute_address = GSIZE_TO_POINTER (op->imm);
+    target.is_indirect = FALSE;
+    target.base = ARM64_REG_INVALID;
+    target.index = ARM64_REG_INVALID;
+    target.disp = 0;
+  }
+  else if (insn->ci->id == ARM64_INS_BLR || insn->ci->id == ARM64_INS_BR)
+  {
+    g_assert (op->type == ARM64_OP_REG);
+
+#if DEBUG_LEVEL == 10
+    g_print ("gum_exec_block_virtualize_branch_insn - BLR & %sconditonal BR\n",
+        is_conditional ? "" : "in");
+    g_print ("\ttarget.base: %d\n", op->reg);
+#endif
+
+    target.is_indirect = FALSE;
+    target.base = op->reg;
+    target.index = ARM64_REG_INVALID;
+    target.disp = 0;
+  }
+  else if(insn->ci->id == ARM64_INS_CBZ || insn->ci->id == ARM64_INS_CBNZ)
+  {
+    op2 = &arm64->operands[1];
+
+    g_assert (op->type == ARM64_OP_REG);
+    g_assert (op2->type == ARM64_OP_IMM);
+
+#if DEBUG_LEVEL == 10
+    if(insn->ci->id == ARM64_INS_CBZ) g_print (
+          "gum_exec_block_virtualize_branch_insn - CBZ\n");
+    else g_print ("gum_exec_block_virtualize_branch_insn - CBNZ\n");
+    g_print ("\ttarget.absolute_address: %d\n", op2->imm);
+#endif
+
+    target.is_indirect = FALSE;
+    target.absolute_address = GSIZE_TO_POINTER (op2->imm);
+    target.base = ARM64_REG_INVALID;
+    target.index = ARM64_REG_INVALID;
+    target.disp = 0;
+  }
+  else if(insn->ci->id == ARM64_INS_TBZ || insn->ci->id == ARM64_INS_TBNZ)
+  {
+    op2 = &arm64->operands[1];
+    op3 = &arm64->operands[2];
+
+    g_assert (op->type == ARM64_OP_REG);
+    g_assert (op2->type == ARM64_OP_IMM);
+    g_assert (op3->type == ARM64_OP_IMM);
+
+#if DEBUG_LEVEL == 10
+    if(insn->ci->id == ARM64_INS_CBZ) g_print (
+          "gum_exec_block_virtualize_branch_insn - CBZ\n");
+    else g_print ("gum_exec_block_virtualize_branch_insn - CBNZ\n");
+    g_print ("\ttarget.absolute_address: %p\n", op3->imm);
+    g_print ("\tbit: %d\n", op2->imm);
+#endif
+
+    target.is_indirect = FALSE;
+    target.absolute_address = GSIZE_TO_POINTER (op3->imm);
+    target.base = ARM64_REG_INVALID;
+    target.index = ARM64_REG_INVALID;
+    target.disp = 0;
+  }
+  else
+  {
+    g_assert_not_reached ();
+  }
+
+  if (insn->ci->id == ARM64_INS_BL || insn->ci->id == ARM64_INS_BLR)
+  {
+#if DEBUG_LEVEL == 10
+    if (insn->ci->id == ARM64_INS_BL)
+      g_print (
+          "gum_exec_block_virtualize_branch_insn - insn->ci->id == ARM64_INS_BL\n");
+    else
+      g_print (
+          "gum_exec_block_virtualize_branch_insn - insn->ci->id == ARM64_INS_BLR\n");
```

Comment:
These exceed the 80 col limit. Also, when a multi-line statement is needed, always use a block.